pkg/spoke/controllers/manifestcontroller/manifestwork_controller.go

```diff
@@ -327,13 +306,20 @@ func (m *ManifestWorkController) applyUnstructured(
 	// Always overwrite required ownerrefs from existing, since ownerrefs of required has been merged to existing
 	required.SetOwnerReferences(existingOwners)
 
+	// Always use generation to check skip for unstructured. It should be fine since resources which do not increment generation
```

Comment:
I'm not clear on why this is true.  I don't know the CR generation incrementing logic offhand.

Reply:
I think generation of CR always get incremented? https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go#L132